doc/DeveloperGuide.md

```diff
@@ -88,6 +88,36 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**Preconditions**
+
+1. Addressbook should have at least one user with tags
+
+**Guarantees**
+
+1. All users with specified tag to be renamed will be renamed
+
+**MSS**
+
+1. User enters rename tag command
+2. User specifies which tag to be renamed and what it should be renamed into
```

Comment:
AddressBook should acknowledge and respond to 1. before 2. happens, so you should add that to the MSS.